src/core/meshcop/commissioner.hpp

```diff
@@ -170,8 +175,9 @@ class Commissioner : public InstanceLocator
      * @param[in]  aTlvs        A pointer to user specific Commissioning Data TLVs.
      * @param[in]  aLength      The length of user specific TLVs in bytes.
      *
-     * @retval OT_ERROR_NONE     Send MGMT_COMMISSIONER_SET successfully.
-     * @retval OT_ERROR_FAILED   Send MGMT_COMMISSIONER_SET fail.
+     * @retval OT_ERROR_NONE           Send MGMT_COMMISSIONER_SET successfully.
+     * @retval OT_ERROR_FAILED         Send MGMT_COMMISSIONER_SET fail.
```

Comment:
Can this return `OT_ERROR_FAILED`?  I believe it can also return `OT_ERROR_NO_BUFS`.